src/resources/views/cytoscape/index.blade.php

```diff
@@ -111,6 +111,19 @@
                                                 'showError' => false,
                                                 ])
                                             </div>
+                                            <div class="col-md-12" style="width:100%">
+                                                @php
+                                                    $relations_list = array_column($used_relations, 'name');
+                                                    $relations = array_combine($relations_list, $relations_list);
```

Comment:
What is happening here

Reply:
Allows me to go from this kind of array [0 => 'Travelled with', 1=> Saw...] to ['Travelled with' => 'Travelled with', 'Saw' => 'Saw']. Otherwise I would be unable to retrieve the name of the relation in index.js